pyhms/tree.py

```diff
@@ -227,3 +229,19 @@ def tree(self) -> str:
             + "\n"
             + format_deme_children_tree(self.root, best_fitness=self.best_individual.fitness)
         )
+
+    def save_animation(self, filepath: str = "hms_tree.mp4") -> None:
```

Comment:
Using this method resulted with:
`ValueError: unknown file extension: .mp4`
When I tried saving to .gif file instead, it worked without issues. Maybe it is a problem on my side, but if .gif format is more errorproof, then maybe it should be used as default file extension?